app/main.py

```diff
@@ -1 +1,22 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parts = command.split()
+    if len(parts) == 3 and parts[0] == "cp":
+        source_file = parts[1]
+        destination_file = parts[2]
+
+        if source_file == destination_file:
+            print("Source and destination file names are the same."
+                  " Nothing to copy.")
```

Comment:
use opposite condition and do nothing if they are the same `It must do nothing in case the user is trying to copy file to file with the same name.`